### PR TITLE
Updated step definitions and added cucumber tests to travis

### PR DESCRIPTION
**Pull request by schreiaj**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -454,6 +454,3 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
-BUNDLED WITH
```
Comment (okeefm) [inline Gemfile.lock]: Do we care if this `BUNDLED WITH` thing gets removed here/added in elsewhere?

File: features/step_definitions/user.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@
   visit '/users/sign_in'
   page.fill_in 'Email', with:  @user.email
   page.fill_in 'Password', with:  @user.password
-  page.click_button 'Log in'
+  page.click_button 'Sign in'
 end
 
 Then(/^the user should see an log in success message$/) do
```
Comment (okeefm) [inline features/step_definitions/user.rb]: Technically it's "a" login message, rather than "an".

Comment (schreiaj) [inline features/step_definitions/user.rb]: Agreed, but I didn't write those specs. I assumed there was a reason to write it that way. Or that it was standard.

Comment (okeefm) [inline features/step_definitions/user.rb]: True.

Comment (ssayer) [inline features/step_definitions/user.rb]: This was copy/pasted initially from a Cucumber/Devise tutorial... it can change

**Pull request merged**